_drafts/mastering-with-git.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Mastering git with team"
+date:   2016-12-6 12:55:28 +0100
+categories: devops project management versioning workflow
+---
+## Introduction
+Work on project with people must be controlled by rules, systems, and best
+pratices work ensure the long life of the project.
+
+Working on project can be addressed to different publics like, a complet project team (private or free projects), 
+simple user who want to contribute to an upstream project, or also core team of a project who want work with external contributors.
+
+This post introduce a complete describe of these use cases and offer to you step by step explaination of these and the 
```

Comment:
a complete description

Reply:
don't you need '-' in use-cases and step-by-step ?

Reply:
and offers you, step-by-step, an explaination ...
or
and explain you, step-by-step, the best practices to deal with them.